kod/object/passive/spell/creaweap.kod

```diff
@@ -21,9 +21,9 @@ resources:
    createweapon_name_rsc = "create weapon"
    createweapon_icon_rsc = icremace.bgf
    createweapon_desc_rsc = \
-      "Materializes a weapon.  Requires nothing but a violent will."
+      "Materializes a weapon. Requires nothing but a violent will."
```

Comment:
I know this is minor and I'm a big time single space guy, but this introduces inconsistencies that I think are better left to a PR that makes a global change.